### ISSUE TITLE
spawn dot ENOENT

### ISSUE THREAD

**Issue by ArtemAdamenko** [ts-graphviz/node#19]
Hello! i try simple example with export dot string to file and to buffer, but have error 
Uncaught Error: spawn dot ENOENT

```ts
// import { digraph, attribute } from "ts-graphviz";
import { exportToFile, exportToBuffer } from "@ts-graphviz/node";
import path from "path";

let dot = 'digraph g { a -> b [label = "Hello World"] }';

await exportToFile(dot, {
  format: "png",
  output: 'example.png',
});

const imageBuffer = await exportToBuffer(dot, {
  format: "png",
});

console.log(imageBuffer)
```

**Comment by kamiazya** [comment]
@ArtemAdamenko 
Thanks, I was able to reproduce it at hand.

It seemed that the internal tmp file handling was failing.

**Comment by kamiazya** [comment]
@all-contributors please add @ArtemAdamenko for bug

**Comment by ArtemAdamenko** [comment]
It turns out that the error exists and will be changed in the future?

**Comment by kamiazya** [comment]
@ArtemAdamenko Yes, I fixed it in #20 and will make a release as 0.4.1 after this.

**Comment by kamiazya** [comment]
@all-contributors please add @ArtemAdamenko for bug
